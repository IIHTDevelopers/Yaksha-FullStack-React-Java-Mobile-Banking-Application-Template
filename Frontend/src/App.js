import React from 'react';

const App = () => {
  return (
    <div>
      Write your logic here
    </div>
  );
};

export default App;
