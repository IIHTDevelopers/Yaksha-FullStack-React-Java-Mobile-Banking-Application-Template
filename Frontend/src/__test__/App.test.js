import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from '../App';

jest.mock('react', () => ({
    ...jest.requireActual('react'),
    useEffect: jest.fn(),
}));

describe('boundary', () => {
    test('AppComponent boundary renders without crashing', () => {
        render(<App />);
    });

    test('AppComponent boundary has "Mobile Banking Application" h1', () => {
        render(<App />);
        expect(screen.queryByText('Mobile Banking Application')).toBeInTheDocument();
    });
});
