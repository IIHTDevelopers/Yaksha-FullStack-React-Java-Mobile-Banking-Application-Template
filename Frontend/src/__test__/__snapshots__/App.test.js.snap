// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`boundary AppTest boundary should mount App without crashing 1`] = `
Array [
  <React.Fragment>
    <UserRegistrationForm />
  </React.Fragment>,
]
`;
